ecommerce-demo/dashboardPlugin/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pluginUrl": "https://gdc-dev-dashboard-plugins.s3.amazonaws.com/internal/dev/dedo/foceasting-plugin/dp_gd_deepar_plugin.js",
+    "pluginName": "gd_deepar",
```

Comment:
Should we name something more related to what the plugin does? Something like `gd_forecasting`. I mean, `deepar` is not self-descriptive and we can't know what the plugin does from its name, can we?

Reply:
After I have the correct Production information, I'll update `pluginUrl` and `pluginName`.